charts/rancher-gatekeeper/rancher-gatekeeper-crd/3.7.0/crd-manifest/assign-customresourcedefinition.yaml

```diff
@@ -0,0 +1,444 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I see some new CRDs like this one. @dhruvmewada15 can you confirm that this is expected based on changes introduced in upstream or should this be excluded?

Reply:
Yes there are few new CRDs added as this is a jump from v3.3.0 to v3.7.0